### ISSUE TITLE
Manual: Calculation of Rw

### ISSUE THREAD

**Issue by k-eks** [YellProgram/Yell#55]
The manual does not state which formula is used for the calculation of the final Rw value.

(I believe it is the square sum of the differences?)

**Comment by aglie** [comment]
Yes, square root of the sum of the square differences multiplied by the
reciprocal_space_multipliers.

ср, 22 мая 2019 г. в 15:44, Gregor Hofer <notifications@github.com>:
